test/verify/check-system-info

```diff
@@ -389,7 +387,7 @@ class TestSystemInfo(MachineCase):
 
         self.login_and_go("/system")
         b.wait_in_text('#system_information_hardware_text', "QEMU")
-        b.click('#system_information_hardware_text')
+        b.click('a:contains("View hardware details")')
```

Comment:
These IDs still exist. Honestly, I'd like to avoid introducing a lot more `:contains` selectors, and use IDs where we have them. `:contains()` is not a standard selector, and each new one will make it harder to eventually get rid of sizzle.

Reply:
`#system_information_hardware_text` is not a link anymore. It's the text only and the link is at the bottom of the card.
See https://garrett.github.io/cockpit-mockweb/navigation2/

The `#system_information_hardware_text` is the Model string and the click is happening on the blue colored link in the bottom of the card.

Reply:
I didn't know that contains is not a standard selector. I can introduce an ID instead for the element if you prefer.

Reply:
Oh, right! Nevermind then. As a middle ground, could at least the cards themselves get IDs (or perhaps they already have unique classes), and then it uses something like `click('#card_system_information a')`?

Reply:
Done.